src/components/update-reminder/update-reminder.scss

```diff
@@ -0,0 +1,113 @@
+@import '../../styles/variables';
+
+.kui-theme--light {
+  --color-header-bg: #{$color-accent-yellow};
+  --color-detail-bg: #{$color-bg-light-4};
+  --color-text: #{$color-bg-light-2};
+  --color-subtext: #{$color-text-light-2};
+}
+
+.kui-theme--dark {
+  --color-header-bg: #{$color-accent-yellow};
+  --color-detail-bg: #{$color-bg-dark-4};
+  --color-text: #{$color-bg-dark-2};
+  --color-subtext: #{$color-text-dark-2};
+}
+
+.update-reminder-unexpanded {
+  height: 32px;
+  width: 296px;
+  position: absolute;
+  right: 0px;
+  top: 0px;
+  background-color: var(--color-header-bg);
+  z-index: 2;
+  font-size: 12px;
+  display: flex;
+  justify-content: space-around;
+
+  p {
+    color: $color-bg-dark-2;
+    margin-top: 6px;
+  }
+
+  button {
+    color: $color-bg-dark-2;
+    border: none;
+    background-color: var(--color-header-bg);
+    font-size: 12px;
+  }
+}
+
+.update-reminder-expanded-header {
+  height: 48px;
+  width: 484px;
+  position: absolute;
+  right: 0px;
+  top: 0px;
+  background-color: var(--color-header-bg);
+  z-index: 2;
+  font-size: 12px;
+  display: flex;
+  justify-content: space-between;
+
+  .close-button-container {
+    padding-right: 20px;
```

Comment:
One thing about this padding is that it makes the close icon clickable when you aren't hovering over the icon. Maybe margin would be better here?